server.js

```diff
@@ -1,38 +1,133 @@
-import express from 'express'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
+import express from "express";
+import cors from "cors";
+import mongoose from "mongoose";
+import data from "./data/netflix-titles.json";
+import listEndpoints from "express-list-endpoints";
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/movies";
+mongoose
+  .connect(mongoUrl, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  })
+  .then((res) => console.log("Connected to DB"))
+  .catch((err) => console.log("Something went wrong!", err));
+mongoose.Promise = Promise;
+
+// Defines the port the app will run on. Defaults to 8080, but can be
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
+
+const Media = mongoose.model("Media", {
+  id: Number,
+  title: String,
+  director: String,
+  cast: String,
+  country: String,
+  dateAdded: String,
+  releaseYear: Number,
+  rating: String,
+  duration: String,
+  categories: String,
+  description: String,
+  type: String,
+});
+
+if (process.env.RESET_DATABASE) {
+  const seedDatabase = async () => {
+    await Media.deleteMany({});
+
+    // adapting the information so that it match the mongoose model I set up.
```

Comment:
Did you use map to change the database instead of changing it directly on the json?